xhr.bs

```diff
@@ -418,7 +416,7 @@ turn causes the <a>use-CORS-preflight flag</a> to be set.)
 <p>To <dfn>terminate the request</dfn>,
 <a for=fetch>terminate</a> the
 <a for=/>fetch</a> algorithm operated by the
-{{XMLHttpRequest}} object with reason <i>fatal</i>.
+{{XMLHttpRequest}} object.
```

Comment:
Maybe inline this algorithm altogether at this point given you're removing 1 of the 3 callsites?